pkg/api/azenvtypes.go

```diff
@@ -99,6 +100,7 @@ var (
 		VnetCNILinuxPluginsDownloadURL:   "https://kubernetesartifacts.azureedge.net/azure-cni/" + AzureCniPluginVerLinux + "/binaries/azure-vnet-cni-linux-amd64-" + AzureCniPluginVerLinux + ".tgz",
 		VnetCNIWindowsPluginsDownloadURL: "https://kubernetesartifacts.azureedge.net/azure-cni/" + AzureCniPluginVerWindows + "/binaries/azure-vnet-cni-windows-amd64-" + AzureCniPluginVerWindows + ".zip",
 		ContainerdDownloadURLBase:        "https://storage.googleapis.com/cri-containerd-release/",
+		CSIProxyDownloadURL:              "https://k8sreleases.blob.core.windows.net/csi-proxy/v0.1.0/binaries/csi-proxy.tar.gz",
```

Comment:
I assume this is a temporary landing spot for this?

Reply:
I will be changing that to https://kubernetesartifacts.azureedge.net/csi-proxy/v0.1.0/binaries/csi-proxy.tar.gz

Reply:
Is that URL not available in mooncake? If it is, we probably don't need to reference it here and can reference it from the cloud env-context-free maps in `pkg/api/k8s_versions.go`

Reply:
The pipeline outputs a mooncake and a non-mooncake artifact so I assume https://kubernetesartifacts.azureedge.net/csi-proxy/v0.1.0/binaries/csi-proxy.tar.gz isn't available in mooncake. Let me follow up on that.

Reply:
@andyzhangx have we mirrored the csi binaries somewhere?

Reply:
@fseldow are you maintaining this?
https://kubernetesartifacts.blob.core.chinacloudapi.cn/csi-proxy/v0.1.0/binaries/csi-proxy.tar.gz

Reply:
https://kubernetesartifacts.blob.core.chinacloudapi.cn/csi-proxy/v0.1.0/binaries/csi-proxy.tar.gz is currently private. We have requested the China team to make it public.